static/app/views/issueDetails/streamline/activitySection.tsx

```diff
@@ -83,6 +82,37 @@ function StreamlinedActivitySection({group}: {group: Group}) {
     [group.activity, mutators, group.id]
   );
 
+  const activities = useMemo(() => {
+    const lastSeenActivity = {
+      type: GroupActivityType.LAST_SEEN,
+      id: uniqueId(),
+      dateCreated: group.lastSeen,
+      project: group.project,
+      data: {},
+    } as GroupActivity;
```

Comment:
avoid casting since the type is fine
```suggestion
    const lastSeenActivity: GroupActivity = {
      type: GroupActivityType.LAST_SEEN,
      id: uniqueId(),
      dateCreated: group.lastSeen,
      project: group.project,
      data: {},
    };
```